recipes-kernel/linux/files/0024-dt-bindings-dp83867-Add-binding-for-the-status-led-c.patch

```diff
@@ -0,0 +1,66 @@
+From 6b3c714413f06cef4035cb7f2e0fb639bf33a3d6 Mon Sep 17 00:00:00 2001
+From: Chao Zeng <chao.zeng@siemens.com>
+Date: Fri, 6 Aug 2021 14:17:23 +0800
+Subject: [PATCH 24/26] dt-bindings:dp83867:Add binding for the status led
+ configuration option
+
+The phy status led of each of board maybe different.
+Provide a method to custom phy status led behavior.
+
+Datasheet:
+http://www.ti.com/product/DP83867IR/datasheet
+
+Signed-off-by: Chao Zeng <chao.zeng@siemens.com>
+---
+ .../devicetree/bindings/net/ti,dp83867.txt     |  5 +++++
+ include/dt-bindings/net/ti-dp83867.h           | 18 ++++++++++++++++++
+ 2 files changed, 23 insertions(+)
+
+diff --git a/Documentation/devicetree/bindings/net/ti,dp83867.txt b/Documentation/devicetree/bindings/net/ti,dp83867.txt
+index 388ff48f53aee..0ded2d97759d6 100644
+--- a/Documentation/devicetree/bindings/net/ti,dp83867.txt
++++ b/Documentation/devicetree/bindings/net/ti,dp83867.txt
+@@ -42,6 +42,11 @@ Optional property:
+ 				    Some MACs work with differential SGMII clock.
+ 				    See data manual for details.
+ 
++	- ti,led-sel    - This configure the status led. See dt-bindings/net/ti-dp83867.h
++					  for different status led settings,select different configurations
++
++	- ti,led-name   - Correspond name of led source.
```

Comment:
Not technically needed, only used for (too verbose) printing.

Reply:
yes .I think this is more clear for user to use this dt-bindings.
how do you think of this modification? need to be removed?

Reply:
Yes, along with the printout. DTs describe hardware and its configuration.

Reply:
I think this  maybe help user to correct configuration for phy status led. 
if we're lack of this description, user can not correspond the led settings to led pin.
So I think keeping this is more convenience  and easy help to debug

Reply:
This will be reject by upstream, so don't even add this here in the first place. Users can correlate the settings at DT level, they don't need printouts.